docs/content/getting-started/production-best-practices.mdx

```diff
@@ -29,7 +29,7 @@ The following list outlines best practices for running OpenFGA in a production e
 
 We recommend:
 
-1. Turn on caching via the flag `--check-query-cache-enabled`. This will reduce latency of requests, but it will increase the staleness of OpenFGA's responses. (The TTL is configurable).
+1. Turn on caching in Check API via flags. This will reduce latency of requests, but it will increase the staleness of OpenFGA's responses. Please see [Cache Expiration](../interacting/consistency.mdx#cache-expiration) for details on the flags.
```

Comment:
Which flags? That should be mentioned here.

Reply:
But I mention the link where the flags are explained 🤔